### PR TITLE
Update Jetpack Search readme

### PR DESCRIPTION
**Pull request by robfelty**
This updates the WordPress.org plugin readme to reflect the new pricing with the free tier.

#### Testing instructions:
Validated the readme with the[ wordpress.org validator](https://wordpress.org/plugins/developers/readme-validator/) - it passed.
I also previewed using [wpreadme](https://wpreadme.com/) and it looks fine there too

#### Does this pull request change what data or activity we track or use?

No it does not.

### CODE REVIEW

File: projects/plugins/search/readme.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-Contributors: automattic, annamcphee, bluefuton, kangzj, jsnmoon
+Contributors: automattic, annamcphee, bluefuton, kangzj, jsnmoon, robfelty, gibrown 
 Tags: search, jetpack
 Requires at least: 6.0
 Requires PHP: 5.6
 Tested up to: 6.1
-Stable tag: 1.2.0
+Stable tag: 1.3.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
```
Comment (gibrown) [inline projects/plugins/search/readme.txt:10]: Let's change the excerpt text also. Right now we aren't using much of the space available: https://wordpress.org/plugins/search/jetpack+search/

Something like:

Easily add instant search and filters to your website or WooCommerce store. Advanced algorithms that boost your search results based on traffic to your site.

File: projects/plugins/search/readme.txt [hunk 2]
```diff
@@ -16,9 +16,9 @@ Incredibly powerful and customizable, Jetpack Search helps your visitors instant
 
 Jetpack Search is an easy-to-use WordPress search plugin that enhances the default site search functionality provided by the CMS. Thanks to advanced site search tools, your visitors will experience the best search experience you can offer.
 
-From a predictive autocomplete algorithm, to instant search results, Jetpack Search has all the features and options for best site search usability and results.
+From real-time search faceting , to instant search results, Jetpack Search has all the features and options for best site search usability and results.
```
Comment (gibrown) [inline projects/plugins/search/readme.txt]: Maybe "filtering and faceting".

I think "facets" are not a very well known term.

File: projects/plugins/search/readme.txt [hunk 3]
```diff
@@ -16,9 +16,9 @@ Incredibly powerful and customizable, Jetpack Search helps your visitors instant
...
 
-Note: A Jetpack Search subscription, or a Jetpack plan subscription that includes Search is required to use this plugin.
+Jetpack Search is free to use for sites with up to 5000 documents and 500 search requests per month. Beyond that, we offer tiered pricing to fit the needs of your site.
```
Comment (gibrown) [inline projects/plugins/search/readme.txt]: "tiered pricing that grows with your site." ???

File: projects/plugins/search/readme.txt [hunk 4]
```diff
@@ -1,10 +1,10 @@
 === Jetpack Search ===
-Contributors: automattic, annamcphee, bluefuton, kangzj, jsnmoon
+Contributors: automattic, annamcphee, bluefuton, kangzj, jsnmoon, robfelty, gibrown 
 Tags: search, jetpack
```
Comment (gibrown) [inline projects/plugins/search/readme.txt]: Remove "jetpack" and add some tags: "filter, woocommerce search, ajax search, product search"

**Review by gibrown — DISMISSED** [review]
Let's also update the author info in https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/search/jetpack-search.php#L4

Similar to what Boost is doing: https://github.com/Automattic/jetpack/blob/a1e5f2edeaf82f5c7021ed39a8b39764ccbd61a4/projects/plugins/boost/jetpack-boost.php

(e.g. "Automattic - Jetpack Search team" )

**Comment by kangzj** [comment]
I'll add these changes in 1.3.0 release, so feel free to close this one 😄

**Comment by robfelty** [comment]
@kangzj @gibrown  - it doesn't look like the readme changes made it into the 1.3.0 release - see https://wordpress.org/plugins/jetpack-search/

**Comment by kangzj** [comment]
Hi @robfelty, I held the readme changes because the new pricing is not live yet. And as discussed in Slack, we are going to release another version of the Search plugin (probably 1.3.1) first thing next week, so let push forward with this PR 😄 Sorry for the confusion.

We might also want to change the plugin description in `plugins/search/jetpack-search.php` as noted by @gibrown .

Thanks for your work.

File: projects/plugins/search/jetpack-search.php [hunk 5]
```diff
@@ -3,7 +3,7 @@
  *
  * Plugin Name: Jetpack Search
  * Plugin URI: https://jetpack.com/search/
- * Description: A cloud-powered replacement for WordPress' search.
+ * Description: Easily add cloud-powered instant search and filters to your website or WooCommerce store with advanced algorithms that boost your search results based on traffic to your site.
  * Version: 1.3.0-alpha
  * Author: Automattic
```
Comment (gibrown) [inline projects/plugins/search/jetpack-search.php]: Change to: Author: Automattic - Jetpack Search team

per p1HpG7-iZ6-p2

**Pull request merged**